renovate.json

```diff
@@ -19,5 +19,10 @@
       ],
       "rangeStrategy": "replace"
     }
-  ]
-}
+  ],
+  "pip_requirements": {
+    "fileMatch": [
+      "^requirements/.*\\.txt$"
+    ]
+  }
+}
```

Comment:
🐼

Reply:
I'm not sure how to interpret that panda.

Reply:
I think it is the panda that watches for trailing newline removals.

Reply:
I noticed the removal, and added it back, but prettier removed it again when I saved the file, so I figured I wouldn't argue with prettier.

Reply:
Can you argue with a panda? 😛

Reply:
Pandas love newlines at the end of files!  (in python you get a linting fail if you don't end a file with a new line)